internal/fs/inode/dir.go

```diff
@@ -1048,6 +1052,33 @@ func (d *dirInode) LocalFileEntries(localFileInodes map[Name]Inode) (localEntrie
 	return
 }
 
+// LOCKS_REQUIRED(fs)
+func (d *dirInode) LocalFileEntriesPlus(localFileInodes map[Name]Inode) (localEntriesPlus map[string]fuseutil.DirentPlus) {
```

Comment:
Do we know, why this method is part of DirInode? To me it seems like a utility to convert one map to another.